packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -244,6 +248,8 @@ class TextEditingController extends ValueNotifier<TextEditingValue> {
   /// [TextEditingController]; **however, one should not also set [selection]
   /// in a separate statement. To change both the [text] and the [selection]
   /// change the controller's [value].**
+  ///
+  /// Setting this will clear the current selection.
```

Comment:
I would make this the last sentence in the previous paragraph since they're related.

Reply:
Also, maybe this is better: "Setting `text` here will clear the current selection and composing range."